### ISSUE TITLE
a11y: Standard procedure and layout for documenting GUIs

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#1146]
First, determine if the GUI in question has reasonable and appropriate accessibility features. Alternatively, determine if the GUI is shown merely for the sake of displaying it as reference, rather how to _use_ the GUI. If either is true, mark the GUI as reviewed (how? TBD) and move to another issue. Bear in mind any images must still be reviewed per #1152.

If the GUI requires additional work, and we cannot resolve it ourselves, then we should carefully document it. The goal of this procedure is to build documentation that parallels the GUI, so the documentation can be used to fill the accessibility gaps missing from the GUI itself.

The following descriptions should be contained in a "rolled up" admonition, perhaps, labeled "How Do I Use The GUI?" We can decide as we go.
 
- Have an image showing the GUI with appropriate alt text.
- Describe the GUI controls in the image.
  - IMAGE ANNOTATIONS
    - Identify significant, related groups of controls, or individual controls. At this stage, treat the identified individual controls as groups when reading further.
    - Add annotations to the image to help identify the groups. Each group should be enclosed and labeled with a number.
    - The numbers should be ordered the same way as keyboard navigation through the GUI.
  - TEXT DESCRIPTION
    - Introduce the purpose of the GUI generally.
    - Introduce the controls with something like the following.

      > The following items describe groups of controls shown in the image. Each group is numbered in keyboard navigation order, matching the labels in the image.

    - Follow with an ordered list, one group of controls per list item.
    - For each group, use the appropriate description template below, group or individual.

If any groups of controls are sufficiently complex, interrelated, or there are more than 7-9 controls in a group, break it out as a separate part of the document. Repeat the process above for just the group broken out.

### Control Description - Groups

> 1. **<GROUP_LABEL>** control group: A text description of the abstract purpose of the controls in the group.

The `<GROUP_LABEL>` should be chosen to match the description, use your best judgment. Keep it short, but meaningful.

The words "control group" are meant to be written literally.

### Control Description - Individual

> 1. **<CONTROL_LABEL>** <CONTROL_TYPE>: A text description of what the control does and how to use it.

The `<CONTROL_LABEL>` is the text displayed visually that is attached to the control. Things like "name" or "email address", etc.

The `<CONTROL_TYPE>` is the type of the control. Things like "button", "text field", or "drop-down menu".

### Examples

TBD
